frontend/src/component/operator-wizard/page-component/Page.js

```diff
@@ -116,7 +116,14 @@ export default class Page extends Component {
     //alert(YAML.safeDump(Dot.object(jsonObject)));
   }
   createResultYaml = jsonObject => {
-    var resultYaml = YAML.safeDump(Dot.object(jsonObject));
+    var resultYaml =
+      "apiVersion: " +
+      document.getElementById("apiVersion").value +
+      "\n" +
+      "kind: " +
+      document.getElementById("kind").value +
+      "\n" +
+      YAML.safeDump(Dot.object(jsonObject));
```

Comment:
@swati-kale @ricardozanini The yaml requires an apiVersion and kind at the top. Not sure if this is the best place to add them, or how to best handle it.

Reply:
Also these two variables are dynamically added to the index.html. I tried making proper ReactJS variables, but had trouble getting that to work.